lib/apple_auth/token.rb

```diff
@@ -75,6 +75,18 @@ def client_urls
       }
     end
 
+    # Apple seems to expect the auth_scheme to be on
+    # the body, the default is :basic_auth
```

Comment:
We can remove this comments and include an explain in the readme file that now we already add the `auth_schema` in the body